src/preact.d.ts

```diff
@@ -5,7 +5,9 @@ declare namespace preact {
 	}
 
 	interface PreactHTMLAttributes {
+		dangerouslySetInnerHTML?:Object;
```

Comment:
@kruczy That seems nice, since anything else in that object would be ignored anyway.